src/assets/Enums.ts

```diff
@@ -32,7 +32,7 @@ export enum HTTPRESPONSE {
     NO_STATUS_UPDATE_REQUIRED = "No status update required",
     NO_EU_VEHICLE_CATEGORY_UPDATE_REQUIRED = "No EU vehicle category update required",
     INVALID_EU_VEHICLE_CATEGORY = "Invalid EU vehicle category",
-    EU_VEHICLE_CATEGORY_MORE_THAN_ONE_TECH_RECORD = "The vehicle has more than one non archived Tech record.",
+    EU_VEHICLE_CATEGORY_MORE_THAN_TWO_TECH_RECORD = "The vehicle has more than two non archived Tech records.",
```

Comment:
I'm being picky, and it's not related to this change, but I think the naming could be improved. Does swapping 'non archived' to 'provisional' sound better? For example, this line would be:
`EU_VEHICLE_CATEGORY_MORE_THAN_TWO_PROVISIONAL_RECORDS = "The vehicle has more than two Provisional Tech records.`

Reply:
Ah, I assumed 'non archived' meant 'provisional'.

Reply:
You'll have to excuse my ignorance of this service. What happens if three records are returned? Won't that error prematurely still?

Reply:
Just read the comments on the Jira ticket. In the ticket description it has:
`If there are 2 provisionals and 1 current tech record then the latest provisional tech record should be shown on the app. `
Is this no longer the case and it should throw an error?

Reply:
It's been confirmed with Steve M that there we are only going to allow 2 non-archived tech records and any more, it should throw the error

Reply:
I'll speak to BA about correcting the description of the story @Ian-Hodges